### PR TITLE
My first Clock Component commited

### PR DESCRIPTION
**Pull request by premwebdeveloper**
I have added a component to show real time watch.

### CODE REVIEW

**Comment by premwebdeveloper** [comment]
This is my first code written in vue js in last 5 years. Achhe marks dena mujhe. Laddu ka dabba ghar pahunch jaega sir.
Thanks

File: src/components/ClockComponent.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { ref, onMounted, onUnmounted } from 'vue';
+
+export default{
+    name:"ClockComponent",
+    data() {
+        return {
+            title:"My Clock (Added by Prem)"
+        }
+    },
+    setup() {
```
Comment (kapilpundit) [inline src/components/ClockComponent.vue:20]: This method setup() is introduced in vue 3.

File: src/components/ClockComponent.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <div>
+        <h2>{{ title }}</h2>
+
+        <div> {{ currentTime }} </div>
+    </div>
+</template>
+
+<script>
+
+import { ref, onMounted, onUnmounted } from 'vue';
```
Comment (kapilpundit) [inline src/components/ClockComponent.vue:11]: All these are Vue 3 Composition Api features.

**Review by kapilpundit — COMMENTED** [review]
PR looks good. Ready for merging into master branch. 
Next thing we want to do is get the interval as a prop from the parent component. Also this prop should have a default value.

**Comment by kapilpundit** [comment]
Merging this into the master branch.

**Pull request merged**